client/src/components/PlayerStatus.jsx

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+var PlayerStatus = ({health}) => {
+  var hearts = [];
+  for(var i = 1; i <=5; i++){
+    if(health>=i*20){
+      hearts.push(<div className="heart-full"></div>);
+    } else {
+      hearts.push(<div className="heart-empty"></div>);
+    }
+  }
+	return (
```

Comment:
indentation is off here :)

And from coding standards, `if(` => `if (` and `for(` => `for (`